### DISCUSSION TITLE
Backend still at 1 despite multiple peers

### DISCUSSION THREAD

**Discussion by duylong** [sni/thruk#1398] [Q&A]
Hi,

Same problem as this issue but solved (https://github.com/sni/Thruk/issues/1268). 

The individual configuration of each peer displays the node but as soon as I add several peers with LMD (use_lmd_core=1), I only see one peer.
The LMD daemon is launched, the conf is loaded with peers.

Do you also have this problem?

**Thruk Version**
Version of Thruk (and OMD if applicable) : 3.18 (LMD 2.2.1)

**Comment by sni** [comment]
using lmd does not change the way you add peers. Using LMD is simply a switch which changes the way backends are handled internally.
So what exactly did you do?

**Comment by duylong** [comment]
My configuration below does not allow me to display my entire cluster?
```
<Component Thruk::Backend>
    <peer>
        name    = DefaultBackend
        type    = livestatus
        <options>
            peer   = node1:6557
            peer   = node2:6557
            peer   = node3:6557
        </options>
    </peer>
</Component>
```

**Comment by sni** [comment]
that's one backend with 3 alternative connections. What you need is something like this:

```
<Component Thruk::Backend>
    <peer>
        name    = node1
        type    = livestatus
        <options>
            peer   = node1:6557
        </options>
    </peer>
    <peer>
        name    = node2
        type    = livestatus
        <options>
            peer   = node2:6557
        </options>
    </peer>
    <peer>
        name    = node3
        type    = livestatus
        <options>
            peer   = node3:6557
        </options>
    </peer>
</Component>
```

**Comment by duylong** [comment]
This is what I was doing before, but I thought LMD aggregates the peers together and Thruk connects to them to retrieve the data.

**Reply by sni** [reply]
Depends on what you want. Most people want to have the backends separated, so they can individualy switch them on/off.
Enabling LMD simply outsources the Backendhandling to a separate, more efficient, external LMD process. Aggregation is still done by Thruk and LMD.

When LMD is enabled, you can also manually access the livestatus socket of LMD where you will get an aggregated data set.

**Reply by sni** [reply]
that's how https://demo.thruk.org uses 2 backends.

**Comment by sni** [comment]
Moving this into discussions, seems there is no issue so far.
